.github/dependabot.yml

```diff
@@ -19,3 +19,9 @@ updates:
     - 7.0.2
     - 7.0.4
     - 7.0.5
+  - package-ecosystem: "github-actions"
```

Comment:
isn't this the wrong indentation here ? You're placing that inside the config of the composer ecosystem

Reply:
Oh yes absolutely, thanks for catching this!